tests/browser/step_definitions/enter-4-digits-bank-account-tax-credits.js

```diff
@@ -14,12 +14,65 @@ Then(
 );
 
 When(
-  "they enter amount and continue from the enter-4-digits-bank-account-tax-credits question page",
+  "they enter correct account number and continue from the enter-4-digits-bank-account-tax-credits question page",
   async function () {
     const singleAmountQuestionPage = new Enter4DigitsBankAccountTaxCredits(
       this.page
     );
-    await singleAmountQuestionPage.answer();
+    await singleAmountQuestionPage.answer("1234");
     await singleAmountQuestionPage.continue();
   }
 );
+
+When(
+  "they not enter account number and continue from the enter-4-digits-bank-account-tax-credits question page",
+  async function () {
+    const singleAmountQuestionPage = new Enter4DigitsBankAccountTaxCredits(
+      this.page
+    );
+    await singleAmountQuestionPage.answer("");
+    await singleAmountQuestionPage.continue();
+  }
+);
+
+When(
+  "they enter invalid account number and continue from the enter-4-digits-bank-account-tax-credits question page",
+  async function () {
+    const singleAmountQuestionPage = new Enter4DigitsBankAccountTaxCredits(
+      this.page
+    );
+    await singleAmountQuestionPage.answer("W123");
+    await singleAmountQuestionPage.continue();
+  }
+);
+
+Then("they should see enter account number error message", async function () {
+  const singleAmountQuestionPage = new Enter4DigitsBankAccountTaxCredits(
+    this.page
+  );
+  const errorMessage = await singleAmountQuestionPage.getErrorMessage();
+
+  expect(singleAmountQuestionPage.isCurrentPage()).to.be.true;
+  expect(singleAmountQuestionPage.hasErrorSummary).to.not.be.false;
+
+  expect(errorMessage).to.equal(
+    "Enter the last 4 digits of your account number"
```

Comment:
I don't think we want to assert on specific content in the tests, might be sufficient just to assert the error summary exists?